pkg/api/client/activities/public_api_service_get_activity_responses.go

```diff
@@ -29,18 +29,6 @@ func (o *PublicAPIServiceGetActivityReader) ReadResponse(response runtime.Client
 			return nil, err
 		}
 		return result, nil
-	case 403:
```

Comment:
do you happen to know the context of why some of these response codes were removed here? https://github.com/tkhq/mono/commit/e5a4673032ff26077c6428651085265d0a3159f4

Reply:
No clue from me... @timurnkey ?

Reply:
I think this was removed to simplify our docs. I don't think this is very useful to single out 403/404s here so I'm fine with letting these go?